polars/lazy/dataframe.ts

```diff
@@ -694,8 +694,8 @@ export const _LazyDataFrame = (_ldf: any): LazyDataFrame => {
           ),
         );
       }
-      let leftOn;
-      let rightOn;
+      let leftOn: any[] | undefined;
+      let rightOn: any[] | undefined;
```

Comment:
I thought I disabled this lint rule in the project?

Reply:
Maybe the newer version of `@biomejs/biome` has different rules?

Reply:
You are correct, the type is not needed, I think VSC was asking for it hence I added it.